Makefile

```diff
@@ -177,42 +177,66 @@ image/build/test:
 test/unit:
 	@TEMPLATE_PATH=$(TEMPLATE_PATH) ./scripts/ci/unit_test.sh
 
 .PHONY: test/e2e/prow
 test/e2e/prow: export component := integreatly-operator
-test/e2e/prow: export OPERATOR_IMAGE := "${IMAGE_FORMAT}"
+test/e2e/prow: export OPERATOR_IMAGE := ${IMAGE_FORMAT}
+test/e2e/prow: export INSTALLATION_TYPE := managed
+test/e2e/prow: export WATCH_NAMESPACE := redhat-rhmi-operator
+test/e2e/prow: export NAMESPACE_PREFIX := redhat-rhmi-
+test/e2e/prow: export INSTALLATION_PREFIX := redhat-rhmi
+test/e2e/prow: export INSTALLATION_NAME := rhmi
+test/e2e/prow: export SKIP_FLAKES := $(SKIP_FLAKES)
 test/e2e/prow: test/e2e
 
 .PHONY: test/e2e/rhoam/prow
 test/e2e/rhoam/prow: export component := integreatly-operator
-test/e2e/rhoam/prow: export MANAGED_API_OPERATOR_IMAGE := "${IMAGE_FORMAT}"
-test/e2e/rhoam/prow: export INSTALLATION_TYPE := "managed-api"
+test/e2e/rhoam/prow: export MANAGED_API_OPERATOR_IMAGE := ${IMAGE_FORMAT}
+test/e2e/rhoam/prow: export INSTALLATION_TYPE := managed-api
+test/e2e/rhoam/prow: export WATCH_NAMESPACE := redhat-rhoam-operator
+test/e2e/rhoam/prow: export NAMESPACE := redhat-rhoam-operator
+test/e2e/rhoam/prow: export NAMESPACE_PREFIX := redhat-rhoam-
+test/e2e/rhoam/prow: export INSTALLATION_PREFIX := redhat-rhoam
+test/e2e/rhoam/prow: export INSTALLATION_NAME := rhoam
+test/e2e/rhoam/prow: export SKIP_FLAKES := $(SKIP_FLAKES)
+test/e2e/rhoam/prow: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/crd deploy/integreatly-rhmi-cr.yml
 test/e2e/rhoam/prow:
-	echo $(OPERATOR_SDK) --verbose test local ./test/e2e --operator-namespace="$(NAMESPACE)" --go-test-flags "-timeout=120m" --debug --image=$(MANAGED_API_OPERATOR_IMAGE)
+	$(OPERATOR_SDK) --verbose test local ./test/e2e --operator-namespace="$(WATCH_NAMESPACE)" --go-test-flags "-timeout=120m" --debug --image=$(MANAGED_API_OPERATOR_IMAGE)
 
 .PHONY: test/e2e
-test/e2e:  export SURF_DEBUG_HEADERS=1
-test/e2e:  cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/crd deploy/integreatly-rhmi-cr.yml
+test/e2e: export SURF_DEBUG_HEADERS=1
+test/e2e: export WATCH_NAMESPACE := $(NAMESPACE)
+test/e2e: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/crd deploy/integreatly-rhmi-cr.yml
 	$(OPERATOR_SDK) --verbose test local ./test/e2e --operator-namespace="$(NAMESPACE)" --go-test-flags "-timeout=120m" --debug --image=$(OPERATOR_IMAGE)
 
 .PHONY: test/e2e/local
+test/e2e/local: export WATCH_NAMESPACE := $(NAMESPACE)
 test/e2e/local: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/crd deploy/integreatly-rhmi-cr.yml
 	$(OPERATOR_SDK) --verbose test local ./test/e2e --watch-namespace="$(NAMESPACE)" --operator-namespace="${NAMESPACE}" --go-test-flags "-timeout=120m" --debug --up-local
 
+.PHONY: test/e2e/regex
+test/e2e/regex: export WATCH_NAMESPACE := $(NAMESPACE)
+test/e2e/regex:
+	go clean -testcache && go test -v ./test/functional -run="$REGEX" -timeout=80m
```

Comment:
I think you are missing brackets around regex
	go clean -testcache && go test -v ./test/functional -run="$(REGEX)" -timeout=80m